### ISSUE TITLE
SpecWorms maintenance

### ISSUE THREAD

**Issue by odontaster** [ices-eg/DIG#538]
A quick check shows around 100 codes that are now different to accepted in WoRMS. What to do about them:
- Leave them all as they are: the ICES subset of WoRMs would not distinguish unaccepted codes. Some databases using this subset count on it holding accepted codes, as I understand it.
- Maintaining it: deprecate unaccepted codes or maybe add a new flag for unaccepted codes different to deprecated. Would allow to be used as Quality Check of data submitted, but means big maintenance effort and a responsibility we do not want to have.
- Slowly move away databases using RECO list as quality check. They could use ERMS database which has flags.

**Comment by Osanna123** [comment]
We're not going to maintain the SpecWoRMS list regarding the accepted/unaccepted flag. We'll add a relevant disclaimer in the long description of the code type.
Inform relevant teams, and suggest using ERMS DB on sql10 for the 'accepted' flag, if relevant. ERMS DB can be implemented as a complete replacement for SpecWoRMS, but it will affect search in Data Portal.

**Comment by odontaster** [comment]
@HjalteParner and @HenrikK-N please be aware of this and let's discuss further options if needed in the respective sprints.

**Comment by HjalteParner** [comment]
... A bit surprised about your take on this!. I think WORMS took over from ERMS ...

**Comment by HenrikK-N** [comment]
It is a problem if the AphiaIds changes over time, so for now from the top of my head it is good, that none of the 'SpecWorm' codes in Vocab are being deprecated. So 100 species out of the 2486 in Vocab are in WoRMS now not accepted?

**Comment by HjalteParner** [comment]
To my knowledge AphiaIDs do not change over time!

What can happen is that an AphiaID can be deprecated, which is quite logic in the taxonomic world (and elsewhere) where as an example one species can be split into 2 or more species etc. when the taxonomists gets more knowledgeable by time.

This is perfectly fine and expected!

In my opinion we should just use the deprecated flag in RECO accordingly. We can setup a routine automatically which do this routinely at a given frequency.

I shouldn't per default have any effects on the respective systems (internally or externally) using this vocabulary.

I do as such not see the issue and do not think we have an issue - except setting up a routines to catch deprecated codes and set the flag accordingly.

**Comment by HenrikK-N** [comment]
Sorry for my wrong formunation AphiaIds do not change over time. But sometimes the AphiaId for an assessment species change I understand. It is not an easy solution to set previously used codes to deprecated in RECO, and let the users deal with the issue, because then the previously used code will not be valid in the RDBES.

**Comment by HjalteParner** [comment]
@HenrikK-N, that's an RDBES issue an not an RECO issue!

RECO users have different use cases, and RDBES should not dictate other systems - if so we will complicate things and not for the good

**Comment by HenrikK-N** [comment]
Hi Hjalte and all
I was just flagging that deprecated AphiaId will create work and make us agree on some standard procedures for updating/re-upload historical data, which include deprecated AphiaIds. But I agree with you, it would be best to follow the code list we have decided to use and the countries uploading also are following. I am sure we can agree on standard procedures for how countries should update/re-upload historical data for deprecated AphiaId. I also talked briefly with the RDBES Core Group, and they also recommended to keep the AphiaId up-to-date, regarding deprecated AphiaId.
